--- gtk-communitheme/Communitheme/gtk-2.0/gtkrc	2018-02-14 11:15:14.918166845 -0700
+++ gtkrc	2018-02-14 11:10:40.759582596 -0700
@@ -16,7 +16,7 @@
 # Foreground/background
 gtk-color-scheme = "fg_color:#3D3D3D\nbg_color:#FAFAFA"
 # Selected foreground/background
-gtk-color-scheme = "selected_fg_color:#ffffff\nselected_bg_color:#E95420"
+gtk-color-scheme = "selected_fg_color:#ffffff\nselected_bg_color:#87A752"
 # Insensitive foreground/background
 gtk-color-scheme = "insensitive_fg_color:#8b8e8f\ninsensitive_bg_color:#f1f1f1"
 # Menus
